src/index.ts

```diff
@@ -219,4 +220,65 @@ app.get("/subjects/:subject_id", async (c) => {
     ),
   });
 });
+
+async function sinceAndUntilSetting(since: string = '2000-01-01T00:00:00', until: string = '3000-01-01T00:00:00'): Promise<any>{
+  const msSince = Date.parse(since);
+  const msUntil = Date.parse(until);
+  if ( isNaN(msSince) || isNaN(msUntil) ){
+    throw new Error("since or until are invalid");
+  }
+  const secondsSince = Math.floor(msSince / 1000);
+  const secondsUntil = Math.floor(msUntil / 1000);
+
+  return { secondsSince: secondsSince, secondsUntil: secondsUntil }
```

Comment:
```suggestion
  return { secondsSince, secondsUntil };
```
格納するキーと変数名が同じなら略記できます.